### ISSUE TITLE
Deprecation warning due to invalid escape sequences

### ISSUE THREAD

**Issue by tirkarthi** [jazzband/django-redshift-backend#65]
### Problem

Deprecation warnings are raised due to invalid escape sequences. This can be fixed by using raw strings or escaping the literals. pyupgrade also helps in automatic conversion : https://github.com/asottile/pyupgrade/

#### Procedure to reproduce the problem

```
find . -iname '*.py' | grep -Ev 'test.py' | xargs -P4 -I{} python3.8 -Wall -m py_compile {}
./django_redshift_backend/base.py:152: DeprecationWarning: invalid escape sequence \(
  m = re.match('varchar\((\d+?)\)', definition)
./django_redshift_backend/base.py:155: DeprecationWarning: invalid escape sequence \(
  'varchar\((\d+?)\)',
```

### Environment info
- OS: <Unix/Linux/Mac/Win/other with version>
- Python version: 3.8
- Django version:
- Django-Redshift-Backend version:

**Comment by browniebroke** [comment]
I think this issue can be closed, fix was merged in #72

**Comment by shimizukawa** [comment]
I think too.

**Comment by shimizukawa** [comment]
Now I've released 2.0.0. Thanks for your contribution!
https://pypi.org/project/django-redshift-backend/2.0.0/
